sass/modules/_alignments.scss

```diff
@@ -16,3 +16,24 @@
 }
 
 // .alignwide {}
+.alignfull,
+.alignwide {
+	width: 100vw;
+	margin-left: calc(50% - 50vw);
+	max-width: none;
+
+	img {
+		width: 100%;
+	}
+}
+
+.alignwide {
+	max-width: 940px;
+
+	@media (min-width: $breakpoint-large) {
```

Comment:
I know it's not a breakpoint, but maybe this should also be `$size__width--main`? Otherwise at 940-960px wide, the content ends up left-aligned.

Reply:
I'm fine with that. My original intention was just to give the `alignwide` a little extra space at 960 so it wasn't just `alignfull` at that point, but I didn't catch the shift.